lib/main.js

```diff
@@ -18,7 +20,11 @@ Object.assign(exports, {
     PlacesProvider.links.init();
     SearchProvider.search.init();
     options.telemetry = false;
-    app = new ActivityStreams(options);
+
+    Task.spawn(function*() {
+      options.clientID = yield ClientID.getClientID();
```

Comment:
Actually, scratch that. Telemetry Client ID is good enough.

@emtwo, please remove the older clientUUID declaration in the same file